Header header
float32 left
float32 right
int32 leftCount
int32 rightCount
